### DISCUSSION TITLE
Call an expr while dubugging

### DISCUSSION THREAD

**Discussion by aaadddfgh** [godotengine/godot-proposals#8337] [Engine Core]
Many debugging tools allow developers run an `expr` at a breakpoint.
Such as GDB can run `call (int)printf("output when break at lines\n")` or `call someIntegerVar=10` when the program stops at a breakpoint. Will developers be able to do similar thing in Godot?

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/4473.
